bin/supplies.js

```diff
@@ -13,12 +13,14 @@ program
   .option('-r, --reference <type>', 'Reference is a unique id for a specific driver; for example a smart contract address')
   .option('-m, --modifiers [addresses]', 'Wallets addresses to subtract from the total supply; for example wallets hold by the foundation or burn addresses. Split the addresses with a comma.')
   .option('-R, --record', 'Record the requests and parameters, and save them as fixtures')
+  .option('-B, --blockchain <type>', 'Define blockchain if driver supports multi-blockchain')
```

Comment:
A bit more descriptive of what this option expects
```suggestion
  .option('-B, --blockchain <type>', 'Issuance blockchain; the blockchain the coin is issued on. This could be their own blockchain for coins like Bitcoin or Monero. Or for example Ethereum for ERC-20 tokens like Basic Attention Token. Providing the blockchain is required if the driver supports multiple blockchains')
```